assignment2.txt

```diff
@@ -0,0 +1,6 @@
+Christopher Liu
+1. Yes, two or more operators can be combined, but relational operators have to be combined with a logical operator like and or or. 
+2. Syntax errors are caused by incorrect usage of variables, functions, or just not following the rules of the program. Usually results in the program crashing. Logic errors are when a program doesn't do what you expect it to do. These are a lot harder to debug as they don't cause the program to immediately stop.
```

Comment:
Correct - note that the difference between the two are that logical errors occur at runtime, whereas syntax errors occur at compilation.